examples/advanced/federated-statistics/hierarchical_stats/demo/visualization.ipynb

```diff
@@ -0,0 +1,222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7c4be7b0",
+   "metadata": {},
+   "source": [
+    "# NVFLARE Hierarchical Statistics Visualization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "987e6028",
+   "metadata": {},
+   "source": [
+    "#### dependencies\n",
+    "\n",
+    "To run examples, you might need to install the dependencies\n",
+    "* ipywidgets\n",
+    "* numpy\n",
+    "* pandas\n",
+    "* matplotlib\n",
+    "* jupyter\n",
+    "\n",
+    "These are captured in the requirements.txt"
```

Comment:
can we do pip install ../requirements.txt as you did in the other notebook?

Reply:
Done, thanks.